setup/air-gapped.md

```diff
@@ -60,6 +71,22 @@ platform images are hosted in Coder's Docker Hub repo.
    docker pull coderenvs/coder-service:<version>
    ```
 
+   Additional images may be needed to configure and run workspaces:
```

Comment:
This line makes it sound like the following images are optional. @mterhar I *think* all of these are required so that they have all Coder features, right?

If these images are required, then we should tell users to install them; if they're not, we should explain when things are required

Reply:
Outside of the ingress controller, all of the listed images are optional, as customers can build their own node & IntelliJ images (for example) and have the option to hook into the OpenVSX extension marketplace. 

That said, while bringing in an _nginx_ ingress controller is optional, I believe we require an ingress in order to access Coder's external IP. So we should clarify that if you are not using an nginx ingress, you'll be responsible for providing your own ingress controller.